src/DynamoCore/Graph/Workspaces/WorkspaceInfo.cs

```diff
@@ -49,6 +49,7 @@ public WorkspaceInfo()
                 double cx = 0;
                 double cy = 0;
                 double zoom = 1.0;
+                double scaleFactor = 1.0;
```

Comment:
@mjkkirschner yes, the scale factor needs to be serialized to the DYN file. This PR takes care of that for desktop. Yes, you would need to introduce the same in Flood as well as the new UI. The tessellated geometry is scaled back after it is returned from ASM and this is taken care of in LibG. Let me know if you have further questions.